src/azure-client.ts

```diff
@@ -0,0 +1,48 @@
+import * as fs from "fs";
+
+import FormData from "form-data";
+
+import { getApiClient } from "./gh-api-client";
+
+export interface Policy {
+  upload_url: string;
```

Comment:
Note that this is deserialized from an earlier HTTP response so we don't control that this is snake case.